src/RestClient/Authenticators/OAuth/src/Internal/Endpoints/DefaultEndpointFactory.cs

```diff
@@ -2,7 +2,7 @@
 {
     using System.Threading.Tasks;
 
-    internal class DefaultEndpointFactory : IAuthenticatorEndpointFactory
+    public class DefaultEndpointFactory : IAuthenticatorEndpointFactory
```

Comment:
these should be moved out of the internal namespace since they are now public

Reply:
Very good point, that's something I didn't notice. I've moved them out.